config/default.yml

```diff
@@ -3,9 +3,10 @@
 Packaging/GemspecGit:
   Description: 'Use pure Ruby alternative instead of `git ls-files`.'
   Enabled: true
-  VersionAdded: '0.86'
+  VersionAdded: '0.0'
```

Comment:
You should check when this cop was first released, I guess `0.1.0`? And if the `0.1.1` was to fix a bug in this cop, then `0.1.1` as `VersionChanged` sounds right :+1:.

Reply:
Okay, one quick question!
It looks like the VersionAdded and VersionChanged should be of the X.Y version format, instead of X.Y.Z (I mean, not really a hard rule, but something other extensions are following).
Do you think we should follow that?

Or X.Y.Z is what we should adopt?

Reply:
What other extensions are doing makes sense, yeah. Go with X.Y.

Reply:
Okay, done from my end^ \o/